index.html

```diff
@@ -1,413 +1,275 @@
+          Spain, but since 2012 he has been living and working in Vienna. When he's not at the office he enjoys a good
+          read, working his way through any delicious recipe, and indulging his love for travelling to new places.
         </p>
-        <p>
-          ZOLLHOF - Tech Incubator<br />
-          Kohlenhofstraße 60<br />
-          90443 Nürnberg<br />
-          Germany<br />
+        <ul class="coach__social">
+          <li>
+            <a href="https://twitter.com/bolonio" target="_blank" rel="noopener noreferrer"
+              title="Follow Adrián on twitter"><i class="icon-twitter-squared"></i></a>
+          </li>
+        </ul>
+      </section>
+
+      <section class="coach">
+        <h3 class="coach__title">Rob Axelson</h3>
+        <img src="images/coach_rob.png" class="coach__image" alt="Rob Axelson" />
+        <p class="coach__bio">
+          Robert Axelsen is a front end developer @ebcont_it, speaker, and open source enthusiast. Most of all, his
+          passion lies with meetups and community, resulting in him being involved with running 5 meetups, 3 of which he
+          founded himself.
         </p>
-        <iframe
-          class="maps"
-          frameborder="0"
-          style="border:0"
-          src="https://www.google.com/maps/embed/v1/place?q=place_id:ChIJd85foHRXn0cRR_JiwviR37M&key=AIzaSyDX5z8zNv_ROMOewRSmyjJ5NeNMEEDJS9M"
-          allowfullscreen
-        ></iframe>
+        <ul class="coach__social">
+          <li>
+            <a href="https://twitter.com/robaxelsen" target="_blank" rel="noopener noreferrer"
+              title="Follow Rob on twitter"><i class="icon-twitter-squared"></i></a>
+          </li>
+        </ul>
       </section>
-      <section>
-        <h2>Code of Conduct</h2>
-        <p>
-          Because this event should be awesome for everybody, we have a
-          <a href="/code-of-conduct">Code of Conduct</a> everyone should obey
-          by.
+
+      <section class="coach">
+        <h3 class="coach__title">Nico Koenig</h3>
+        <img src="images/coach_nico.png" class="coach__image" alt="Nico Koenig" />
+        <p class="coach__bio">
+          Nico is a freelancer and trainer for HTML, CSS & JavaScript. In his
+          non-existing free time, Nico organizes the Franken JS meetup and
+          brews some kickass hand ground coffee.
         </p>
+        <ul class="coach__social">
+          <li>
+            <a href="https://twitter.com/theNicoKoenig" target="_blank" rel="noopener noreferrer"
+              title="Follow Nico on twitter"><i class="icon-twitter-squared"></i></a>
+          </li>
+        </ul>
       </section>
-    </main>
-    <footer></footer>
-  </body>
+    </article>
+
+    <article>
+      <h2>Code of Conduct</h2>
+      <p>
+        Because this event should be awesome for everybody, we have a
+        <a href="/code-of-conduct">Code of Conduct</a> everyone should obey
```

Comment:
Super minor thing here with "obey by". So I tried looking this up, couldn't quite find anything, but I'm not sure it sounds right, grammatically. 

I'm not (okay nearly) a native speaker but think that "abide by" sounds better. What do you think?

As said, no big deal, just noticed it :)

Reply:
In the COC itself, we already use "abide by".

Changed the wording in the index.html

Reply:
Ahh, I missed that, my bad! Thanks for taking a look :)